tests/test_checkout_integration.py

```diff
@@ -7,6 +8,27 @@
 from saleor.order.models import Order
 from saleor.shipping.models import ShippingMethod
 
+# Resolve urls
+urls = {
+    'cart': reverse('cart:index'),
+    'checkout_index': reverse('checkout:index'),
+    'checkout_shipping_address': reverse('checkout:shipping-address'),
+    'checkout_shipping_method': reverse('checkout:shipping-method'),
+    'checkout_summary': reverse('checkout:summary')}
```

Comment:
These don't help at all. `urls['cart']` is no better than `reverse('cart:index')` and requires you to constantly check what URL is being referenced.